src/content/docs/logs/ui-data/parsing.mdx

```diff
@@ -96,98 +96,175 @@ Here's an overview of how New Relic implements parsing of logs:
-      </th>
-      <th>
-        Type stored in the New Relic database
-      </th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>
-        `boolean`
-      </td>
-      <td>
-        `boolean`
-      </td>
-    </tr>
-    <tr>
-      <td>
-        `byte`
-        `short`
-        `int`
-        `integer`
-      </td>
-      <td>
-        `integer`
-      </td>
-    </tr>
-    <tr>
-      <td>
-        `long`
-      </td>
-      <td>
-        `long`
-      </td>
-    </tr>
-    <tr>
-      <td>
-        `float`
-      </td>
-      <td>
-        `float`
-      </td>
-    </tr>
-    <tr>
-      <td>
-        `double`
-      </td>
-      <td>
-        `double`
-      </td>
-    </tr>
-    <tr>
-      <td>
-        `string` (default)
-        `text`
-      </td>
-      <td>
-        `string`
-      </td>
-    </tr>
-    <tr>
-      <td>
-        `date`
-        `datetime`
-      </td>
-      <td>
-        ISO 8601 time as a `long`
-      </td>
-    </tr>
-  </tbody>
-</table>
+  This information is accurate, but it's not exactly intuitive what it means. Grok patterns help you extract and understand the telemetry data you want. For example, a log record like this is much easier to use:
+
+  ```
+  {
+    "host_ip": "43.3.120.2",
+    "bytes_received": 2048,
+    "bytes_sent": 0
+  }
+  ```
+
+  To do this, create a Grok pattern that extracts these three fields; for example:
 
-See [our blog post](https://newrelic.com/blog/how-to-relic/how-to-use-grok-log-parsing) for more details.
+  ```
+  "%{IP:host_ip} %{INT:bytes_received} %{INT:bytes_sent}"
+  ```
+  
+  After processing, your log record will include the fields `host_ip`, `bytes_received`, and `bytes_sent`. Now you can use these fields in New Relic to filter, facet, and perform statistical operations on your log data. For more details about how to parse logs with Grok patterns in New Relic, see [our blog post](https://newrelic.com/blog/how-to-relic/how-to-use-grok-log-parsing).
+
+  </Collapser>
+
+  <Collapser
+    id="grok-ui"
+    title="UI example: Creating a Grok parsing rule"
+  >
+
+  If you have the correct permissions, you can create parsing rules in our UI to create, test, and enable Grok parsing. For example, to get a specific type of error message for one of your microservices called Inventory Services, you would create a Grok parsing rule that looks for a specific error message and product. To do this:
+  
+  1. Give the rule a name; for example, `Inventory Services error parsing`.
+  2. Identify the attribute/value pair that acts as a pre-filter for the incoming logs; for example, `entity.name` and the value `Inventory Service`. This pre-filter narrows down the number of logs that need to be processed by your rule, removing unnecessary processing.
+  3. Add the Grok parsing rule; for example:
+
+    ```
+    Inventory error: %{DATA:error_message} for product %{INT:product_id}
+    ```
+
+    Where:
+
+    * `Inventory error` = Your parsing rule's name
```

Comment:
Just a tiny consistency comment, we usually use : instead of = for this.

Reply:
d'oh! You're right! I'll fix. Thanks for catching that!